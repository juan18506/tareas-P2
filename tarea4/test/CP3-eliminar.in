# implementar eliminarPriorotaria

# creo la cola de tamaño 10
crearColaPrioridad 10

# elimino cuando la cola está vacía
eliminarPersonaPrioritaria

# creo Persona 8 y agrego a cola
crearAgenda
crearPersona 8 10 Persona8
crearFecha 10/10/2021
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# verifico que la persona 8 está en la cola
estaPersonaEnColaPrioridad 8

# la persona 8 es la única y la prioritaria
imprimirPersonaPrioritaria

# elimino la persona 8 de la cola
eliminarPersonaPrioritaria

# ahora no debería estar y la cola es vacía
estaPersonaEnColaPrioridad 8
estaVaciaColaPrioridad

# creo Persona 7 y agrego a cola
crearAgenda
crearPersona 7 10 Persona7
crearFecha 7/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 10 y agrego a cola
crearAgenda
crearPersona 10 10 Persona10
crearFecha 11/10/2021
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 1 y agrego a cola
crearAgenda
crearPersona 1 10 Persona1
crearFecha 12/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 2 y agrego a cola
crearAgenda
crearPersona 2 10 Persona2
crearFecha 2/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 8 y agrego a cola
crearAgenda
crearPersona 8 10 Persona8
crearFecha 10/10/2021
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 9 y agrego a cola
crearAgenda
crearPersona 9 10 Persona9
crearFecha 9/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 5 y agrego a cola
crearAgenda
crearPersona 5 10 Persona5
crearFecha 5/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 3 y agrego a cola
crearAgenda
crearPersona 3 10 Persona3
crearFecha 3/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 4 y agrego a cola
crearAgenda
crearPersona 4 10 Persona4
crearFecha 4/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# la persona prioritaria sigue siendo 8
imprimirPersonaPrioritaria

# ahora la elimino y la prioritaria es la 10
eliminarPersonaPrioritaria
imprimirPersonaPrioritaria

# vuelvo a eliminar y la prioritaria es la 2
eliminarPersonaPrioritaria
imprimirPersonaPrioritaria


# creo Persona 6 y agrego a cola
crearAgenda
crearPersona 6 10 Persona6
crearFecha 6/6/6
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 8 y agrego a cola
crearAgenda
crearPersona 8 10 Persona8
crearFecha 10/10/2021
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 10 y agrego a cola
crearAgenda
crearPersona 10 10 Persona10
crearFecha 10/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# ahora la 6 es la prioritaria
imprimirPersonaPrioritaria

# vuelvo a eliminar y la prioritaria es la 8
eliminarPersonaPrioritaria
imprimirPersonaPrioritaria

# vuelvo a eliminar y la prioritaria es la 2
eliminarPersonaPrioritaria
imprimirPersonaPrioritaria

eliminarPersonaPrioritaria
eliminarPersonaPrioritaria
eliminarPersonaPrioritaria
eliminarPersonaPrioritaria
eliminarPersonaPrioritaria
eliminarPersonaPrioritaria
eliminarPersonaPrioritaria

# siguen quedando elementos en la cola
estaVaciaColaPrioridad

# la prioritaria es la persona 1
imprimirPersonaPrioritaria

# y su prioridad es la fecha:
imprimirPrioridadPersona 1

# elimino y ahora si queda vacía
eliminarPersonaPrioritaria
estaVaciaColaPrioridad

liberarColaPrioridad

Fin