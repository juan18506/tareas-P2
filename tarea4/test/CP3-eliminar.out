1>#  implementar eliminarPriorotaria.
2>#  creo la cola de tamaño 10.
3>4>#  elimino cuando la cola está vacía.
5>6>#  creo Persona 8 y agrego a cola.
7>8>9>10>11>12>Persona 8: Persona8, 10 años
Evento 1:  Evento1
Fecha: 10/10/2021
13>14>#  verifico que la persona 8 está en la cola.
15>La persona 8 está en la cola de prioridad.
16>#  la persona 8 es la única y la prioritaria.
17>Persona 8: Persona8, 10 años
Evento 1:  Evento1
Fecha: 10/10/2021
18>#  elimino la persona 8 de la cola.
19>20>#  ahora no debería estar y la cola es vacía.
21>La persona 8 no está en la cola de prioridad.
22>La cola de prioridad está vacía.
23>#  creo Persona 7 y agrego a cola.
24>25>26>27>28>29>Persona 7: Persona7, 10 años
Evento 1:  Evento1
Fecha: 7/10/2022
30>31>#  creo Persona 10 y agrego a cola.
32>33>34>35>36>37>Persona 10: Persona10, 10 años
Evento 1:  Evento1
Fecha: 11/10/2021
38>39>#  creo Persona 1 y agrego a cola.
40>41>42>43>44>45>Persona 1: Persona1, 10 años
Evento 1:  Evento1
Fecha: 12/10/2022
46>47>#  creo Persona 2 y agrego a cola.
48>49>50>51>52>53>Persona 2: Persona2, 10 años
Evento 1:  Evento1
Fecha: 2/10/2022
54>55>#  creo Persona 8 y agrego a cola.
56>57>58>59>60>61>Persona 8: Persona8, 10 años
Evento 1:  Evento1
Fecha: 10/10/2021
62>63>#  creo Persona 9 y agrego a cola.
64>65>66>67>68>69>Persona 9: Persona9, 10 años
Evento 1:  Evento1
Fecha: 9/10/2022
70>71>#  creo Persona 5 y agrego a cola.
72>73>74>75>76>77>Persona 5: Persona5, 10 años
Evento 1:  Evento1
Fecha: 5/10/2022
78>79>#  creo Persona 3 y agrego a cola.
80>81>82>83>84>85>Persona 3: Persona3, 10 años
Evento 1:  Evento1
Fecha: 3/10/2022
86>87>#  creo Persona 4 y agrego a cola.
88>89>90>91>92>93>Persona 4: Persona4, 10 años
Evento 1:  Evento1
Fecha: 4/10/2022
94>95>#  la persona prioritaria sigue siendo 8.
96>Persona 8: Persona8, 10 años
Evento 1:  Evento1
Fecha: 10/10/2021
97>#  ahora la elimino y la prioritaria es la 10.
98>99>Persona 10: Persona10, 10 años
Evento 1:  Evento1
Fecha: 11/10/2021
100>#  vuelvo a eliminar y la prioritaria es la 2.
101>102>Persona 2: Persona2, 10 años
Evento 1:  Evento1
Fecha: 2/10/2022
103>#  creo Persona 6 y agrego a cola.
104>105>106>107>108>109>Persona 6: Persona6, 10 años
Evento 1:  Evento1
Fecha: 6/6/6
110>111>#  creo Persona 8 y agrego a cola.
112>113>114>115>116>117>Persona 8: Persona8, 10 años
Evento 1:  Evento1
Fecha: 10/10/2021
118>119>#  creo Persona 10 y agrego a cola.
120>121>122>123>124>125>Persona 10: Persona10, 10 años
Evento 1:  Evento1
Fecha: 10/10/2022
126>127>#  ahora la 6 es la prioritaria.
128>Persona 6: Persona6, 10 años
Evento 1:  Evento1
Fecha: 6/6/6
129>#  vuelvo a eliminar y la prioritaria es la 8.
130>131>Persona 8: Persona8, 10 años
Evento 1:  Evento1
Fecha: 10/10/2021
132>#  vuelvo a eliminar y la prioritaria es la 2.
133>134>Persona 2: Persona2, 10 años
Evento 1:  Evento1
Fecha: 2/10/2022
135>136>137>138>139>140>141>142>#  siguen quedando elementos en la cola.
143>La cola de prioridad no está vacía.
144>#  la prioritaria es la persona 1.
145>Persona 1: Persona1, 10 años
Evento 1:  Evento1
Fecha: 12/10/2022
146>#  y su prioridad es la fecha:.
147>12/10/2022
148>#  elimino y ahora si queda vacía.
149>150>La cola de prioridad está vacía.
151>152>Fin.
